src/index.js

```diff
@@ -0,0 +1,195 @@
+  } else if (skyValue === "4") {
+    state.skyField.textContent = skyOptions.rainy;
+  } else if (skyValue === "5") {
+    state.skyField.textContent = skyOptions.snowy;
+  } else {
+    state.skyField.textContent = "";
+  }
+};
+
+const weatherOptions = {
+  Clouds: "3",
+  Haze: "3",
+  Mist: "3",
+  Clear: "2",
+  Rain: "4",
+  Drizzle: "4",
+  Thunderstorm: "4",
+  Snow: "5",
+};
+
+const state = {
+  cityTextField: null,
+  headerCity: null,
+  cityResetButton: null,
+  landscapeField: null,
+  skyField: null,
+  skyOptions: null,
+  currTempButton: null,
+  temperatureValue: null,
+  currentTemperature: 50,
+  temperatureIncreaseButton: null,
+  temperatureDecreaseButton: null,
+};
+
+const getTemperatureColor = (temperature) => {
+  if (temperature >= 80) {
+    return "red";
+  } else if (temperature >= 70 && temperature <= 79) {
+    return "orange";
+  } else if (temperature >= 60 && temperature <= 69) {
+    return "yellow";
+  } else if (temperature >= 50 && temperature <= 59) {
+    return "green";
+  } else if (temperature <= 49) {
+    return "teal";
+  }
+};
+
+const tempColorUpdate = () => {
+  state.temperatureValue.className = getTemperatureColor(
+    state.currentTemperature
+  );
+};
+
+const increaseTemperature = () => {
+  state.currentTemperature += 1;
+  state.temperatureValue.textContent = state.currentTemperature;
+  allGraphicsUpdate();
+};
+
+const decreaseTemperature = () => {
+  state.currentTemperature -= 1;
+  state.temperatureValue.textContent = state.currentTemperature;
+  allGraphicsUpdate();
+};
+
+const locUrl = "http://localhost:5000/location";
+const weatherUrl = "http://localhost:5000/weather";
+
+const getTempFromLocation = () => {
+  const q = state.cityTextField.value;
+  axios
+    .get(locUrl, { params: { q } })
+    .then((response) => {
+      const lat = response.data[0].lat;
+      const lon = response.data[0].lon;
+      axios
+        .get(weatherUrl, { params: { lat, lon } })
+        .then((weatherResponse) => {
+          const tempCelsius = weatherResponse.data.main.temp - 273.15;
+          const newTemp = Math.floor(tempCelsius * (9 / 5) + 32);
+          state.currentTemperature = newTemp;
+          state.temperatureValue.textContent = state.currentTemperature;
+          const mainWeather = weatherResponse.data.weather[0].main;
+          console.log(mainWeather);
+          if (mainWeather in weatherOptions) {
+            state.skyOptions.value = weatherOptions[mainWeather];
+            console.log(weatherOptions[mainWeather]);
+          } else {
+            state.skyOptions.value = 1;
+          }
+
+          allGraphicsUpdate();
+        })
+        .catch((error) => {
+          console.log("Weather call failed", error);
+        });
+    })
+    .catch((error) => {
+      console.log("Location call failed", error);
+    });
+};
+
+const allGraphicsUpdate = () => {
+  tempColorUpdate();
+  landscapeUpdate();
+  skyOptionsUpdate();
+};
+
+const loadData = () => {
```

Comment:
This function is looking up UI elements, so rather than `loadData`, consider a name like `resolveUIElements`.